docs/features/teams.rst

```diff
@@ -0,0 +1,85 @@
+.. _teams:
+
+Teams
+=====
+
+Pootle added the team page which is meant to ease management of team members
```

Comment:
Not a history lesson, state what it does not that its new or anything.

"Pootle's team page is designed to assist in managing teams."